experiments/AMIP/modular/cli_options.jl

```diff
@@ -27,6 +27,10 @@ function argparse_settings()
         help = "Boolean flag indicating whether (1st order) monotone and conservative remapping is applied."
         arg_type = Bool
         default = false
+        "--turb_flux_partition"
+        help = "Method to partition turbulent fluxes. [`PartitionedComponentModelGrid`, `CombinedAtmosGrid`]"
```

Comment:
Could be `PartitionedComponentGrid` instead of `PartitionedComponentModelGrid` for brevity

Reply:
yes, as agreed offline 👍